image_proc/src/crop_decimate.cpp

```diff
@@ -131,9 +131,9 @@ void CropDecimateNode::imageCb(
 {
   /// @todo Check image dimensions match info_msg
 
-  if (pub_.getNumSubscribers() < 1) {
-    return;
-  }
+  //if (pub_.getNumSubscribers() < 1) {
+    //return;
+  //}
```

Comment:
We want this code to be here. It bypasses publishing unless at least one subscriber is present on the topic.